### PR TITLE
Feature/24662 aa timealert numpad settings

### PR DESCRIPTION
**Pull request by danwilliams**
Pupils need to be able to confirm the access arrangements applied.

1. If audible time alert and/or removal of number pad (access arrangement setting) has been set for a pupil, when the pupil signs in, then display your settings page
2. Pupil can view 'Your Settings' page as per prototype
  - Pupil can view audible time alert and/or removal of number pad in alphabetical order
  - Pupil can view 'next' button, if next is selected then display welcome page
      - Pupil can view welcome page as per prototype
  - Pupil can view 'sign out' link, If the link is selected then the pupil is signed out

### CODE REVIEW

**Comment by GuyHarwood** [comment]
@danwilliams would you mind adding a brief description at the top of the PR to explain what the feature does please.

**Comment by danwilliams** [comment]
@GuyHarwood Done.

File: test/pupil/features/step_definitions/access_arrangement_steps.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  expect(access_arrangements_setting_page).to have_sign_out
+  expect(access_arrangements_setting_page).to have_next_btn
+  expect(access_arrangements_setting_page).to have_access_arrangements_list
+end
+
+Then(/^I can see following access arrangement$/) do |table|
+expect(access_arrangements_setting_page.access_arrangements_list[0].text.eql?(table.hashes[0]['access_arrangement_type'])).to be_truthy, "Expected: #{table.hashes[0]['access_arrangement_type']}....Got: #{access_arrangements_setting_page.access_arrangements_list[0].text}"
+expect(access_arrangements_setting_page.access_arrangements_list[1].text.eql?(table.hashes[1]['access_arrangement_type'])).to be_truthy, "Expected: #{table.hashes[1]['access_arrangement_type']}....Got: #{access_arrangements_setting_page.access_arrangements_list[1].text}"
+
+end
```
Comment (activemq) [inline test/pupil/features/step_definitions/access_arrangement_steps.rb]: Can we format this step please @sinhaamit1980

File: test/pupil/features/step_definitions/access_arrangement_steps.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      when 'Audible Time Alert'
+        SqlDbHelper.set_pupil_access_arrangement(@pupil['id'], new_time, new_time, 1)
+      when 'Remove on-screen number pad'
+        SqlDbHelper.set_pupil_access_arrangement(@pupil['id'], new_time, new_time, 7)
+    end
+  end
+
+  sign_in_page.login(@school['pin'], @pin)
+  sign_in_page.sign_in_button.click
+end
```
Comment (activemq) [inline test/pupil/features/step_definitions/access_arrangement_steps.rb]: Can any of the above be pulled out in to a method? @sinhaamit1980

**Pull request merged**